support/cas-server-support-cassandra-ticket-registry/src/main/java/org/apereo/cas/config/CassandraProperties.java

```diff
@@ -0,0 +1,80 @@
+package org.apereo.cas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * This is {@link CassandraProperties}.
+ *
+ * @author David Rodriguez
+ * @since 5.1.0
+ */
+@ConfigurationProperties("cassandra")
+public class CassandraProperties {
+
+    private String contactPoints;
+    private String username;
+    private String password;
+    private String tgtTable;
+    private String stTable;
+    private String expiryTable;
+    private String lastRunTable;
+
+    public String getContactPoints() {
+        return contactPoints;
+    }
+
+    public void setContactPoints(final String contactPoints) {
+        this.contactPoints = contactPoints;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(final String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(final String password) {
+        this.password = password;
+    }
+
+    public String getTgtTable() {
+        return tgtTable;
+    }
+
+    public void setTgtTable(final String tgtTable) {
```

Comment:
I totally agree with Misagh. The schema, keyspace, etc. should all be treated as internal implementation detail and not "leaked" into deployer's configuration space